src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,90 @@
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+
+import { TableRow } from '../TableRow/TableRow';
+import { Loader } from '../Loader';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loader, setLoader] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('');
+
+  useEffect(() => {
+    setLoader(true);
+    getPeople()
+      .then(setPeople)
+      .catch((error) => {
+        setLoader(false);
+        setErrorMessage('Something went wrong');
+        throw error;
+      })
+      .finally(() => setLoader(false));
+  }, []);
+
+  const peopleFromServer = people.map(person => ({
+    ...person,
+    mother: people.find(p => p.name === person.motherName),
```

Comment:
looks very similar, it seems to me that you can create separated function for `.find` logic